manim/utils/polylabel.py

```diff
@@ -45,14 +52,14 @@ def __init__(self, rings: Sequence[Point3D_Array]) -> None:
             cy = np.sum((y + yr) * factor) / (6.0 * self.area)
             self.centroid = np.array([cx, cy])
 
-    def compute_distance(self, point: Point3D) -> float:
+    def compute_distance(self, point: Point2DLike) -> float:
```

Comment:
I think it might be `Point2D`, because of the `point - self.start` line?

Reply:
`self.start` is a NumPy array (a `Point2D_Array`), so this is fine. Subtracting a NumPy array from a tuple gives another NumPy array:
```pycon
>>> (3, 2) - np.array([[1, 1], [2, 2]])
array([[2, 1],
       [1, 0]])
```

Reply:
Oh good to know :)

Reply:
Oh good to know :)